pages/index.tsx

```diff
@@ -1,5 +1,6 @@
 import { Flex, Heading, Text } from "@chakra-ui/react";
 import Head from "next/head";
+import React from "react";
```

Comment:
Just curious, are we going to have to do this on every page?

Reply:
I don't think this is even a required change